settings.gradle.kts

```diff
@@ -6,6 +6,7 @@ include(":app:aem:actions.checks")
 include(":app:aem:actions.main")
 include(":app:aem:ui.apps")
 include(":app:aem:ui.content")
+include(":app:aem:ui.react")
```

Comment:
You should rename this module to `ui.frontend` as it is named in adobe's archetype.